owners/src/github.js

```diff
@@ -65,12 +65,12 @@ class PullRequest {
   async processOpened() {
     const prInfo = await this.getMeta();
     // TODO: Revieweres here is to be assigned to the Pull Request.
-    /* eslint no-unused-vars: 0 */
+    /* eslint-disable no-unused-vars */
```

Comment:
Can this be changed to `// eslint-disable-line no-unused-vars` or `// eslint-disable-next-line no-unused-vars`?

Reply:
Done.